### ISSUE TITLE
Suggestion: Allow the colour of the text to be changed

### ISSUE THREAD

**Issue by sbrl** [owncloud/documents#607]
Having the ability to change the colour of the text when editing would make this already awesome app even more useful - I find colour coding things helps me a lot.

**Comment by RealRancor** [comment]
@sbrl The documents app is based on https://github.com/kogmbh/WebODF/issues. Try to create a feature request upstream: https://github.com/kogmbh/WebODF/issues to get such a feature included.

**Comment by sbrl** [comment]
@RealRancor Right. I've found that someone has already opened an issue for this over at kogmbh/WebODF#821, for anyone else who is interested.

Perhaps this issue should be labelled upstream then.

**Comment by 0ki** [comment]
I think it is already possible.
Format -> Paragraph -> Font Effects -> Color -> Text

**Comment by sbrl** [comment]
@0ki I didn't see that! Yes it does seem to be possible already. I do think though that a button is needed - nobody wants to go diving that deep into a GUI just to change the font colour.
